src/pages/admin/MySettings/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment, useRef } from 'react';
+import { Tab } from '@headlessui/react';
+import Breadcrumb from '../../components/shared/Breadcrumb';
+import AdminProfileTab from './Profile';
+import AdminAccountTab from './Account';
+
+const classNames = (...classes) => classes.filter(Boolean).join(' ');
+
+const TabNames = ['Profile', 'Account'];
+
+const AdminMySettings = () => {
+  const fileInput = useRef(null);
+  return (
+    <Fragment>
+      <div className='mb-5 flex text-brown-900'>
+        <Breadcrumb text='Dashboard' to='/admin/dashboard' />
+        <Breadcrumb text='My Settings' last />
+      </div>
+      <section>
+        <div className='text-2xl font-bold text-brown-900'>My Settings</div>
+        <hr className='mt-3 border border-orange-900' />
+        <div className='mx-8 mt-8 flex'>
+          <div className='flex flex-col items-center'>
+            <img
```

Comment:
```suggestion
            <LazyLoadImage
```
Can we use LazyLoadImage?